tests/testthat/test-compare.R

```diff
@@ -11,3 +11,60 @@ with_mock_dir(test_path("logged-in"), {
     )
   })
 })
+
+
+with_mock_dir(test_path("resp-class"), {
+  test_that("zscore and pairwise thresholds return correct values", {
+    z_thres <- 10
+    percent_threshold <- 20
+
+    resp <- compare(
+      type = "runs",
+      "ee1",
+      "f00d",
```

Comment:
I am unable to comprehend this bit, a comment would be great

Reply:
Ah. These are simply hex words. Have a look at this: https://nedbatchelder.com/text/hexwords.html